src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -5,20 +5,19 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
-  This handbook helps contributors to web.dev create effective, engaging content
-  and get it published as easily as possible.
+  Learn how to write for an international audience with varying needs and preferences.
 ---
 
-web.dev has a large international audience. Members of that audience should be able to access and understand web.dev content, and they should feel included in the ways we speak to our readers.
+web.dev has a large international audience. We want members of that audience to access and understand web.dev content, and they should feel included in the ways we speak to our readers.
 
 ## Use readable language
-Aim for a grade 8 reading level when possible. If you don't have a feel for what grade 8 texts look like, check out some [examples on Newsela](https://newsela.com/articles/#/rule/latest?grade_levels=8.0). If you're feeling frisky, you can also run your text through a readability test.
-* [TextEvaluator](http://textevaluator.ets.org/TextEvaluator/) can be used for pieces up to 1,600 words.
-* You can also use [Lexile](https://lexile.com/educators/tools-to-support-reading-at-school/tools-to-determine-a-books-complexity/the-lexile-analyzer/) for pieces up to 1,000 words.
+Aim for a grade 8 reading level when possible. If you don't have a feel for what grade 8 texts look like, check out some [examples on Newsela](https://newsela.com/articles/#/rule/latest?grade_levels=8.0). You can also run your text through a readability test.
+* For pieces up to 1,600 words, use [TextEvaluator](http://textevaluator.ets.org/TextEvaluator/).
```

Comment:
This phrasing changes the intended meaning. I prefer people to use TextEvaluator because it's a better tool. However, if they want a second data point, they can also use Lexile. Then again, that's probably unlikely, so maybe I can just cut the Lexile bullet.

Reply:
See my notes above about reading level.